automatic/PDFCreator/PDFCreator.nuspec

```diff
@@ -17,8 +17,4 @@ PDFCreator – PDFCreator easily creates PDFs from any Windows program. Use it l
     <tags>pdf admin</tags>
     <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages</packageSourceUrl>
```

Comment:
Its also good chance (although not mandatory) to update other metadata:
- Give some features
- Add some more tags such as `editor`, `office` etc.
- Add docs, bugtrackerurl, mailinglist etc.